public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -54,6 +54,7 @@ data class EntitlementInfo(
     val billingIssueDetectedAt: Date?,
     val ownershipType: OwnershipType,
     private val jsonObject: JSONObject,
+    @ExperimentalPreviewRevenueCatPurchasesAPI
```

Comment:
Originally I tried to add this annotation to the `VerificationResult` and `EntitlementVerificationMode` enums themselves, but that made it so the whole class required the annotation. So in the end I only added to the APIs where those enums may be used.

Reply:
Yeah, this is due to [`RequiresOptIn` Contagiousness](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-requires-opt-in/#contagiousness).